CharlieBackend.Data/Repositories/Impl/CourseRepository.cs

```diff
@@ -37,6 +37,14 @@ public async Task<bool> IsCourseHasGroupAsync(long id)
             return await _applicationContext.StudentGroups.AnyAsync(s => s.CourseId == id);
         }
 
+        public async Task<IList<Course>> GetAllActiveCourses()
+        {
+            return await _applicationContext.Courses
+                .Where(x => x.IsActive == true)
+                .Select(x => x)
```

Comment:
this query can be simplified, due to type of the parameter isActive is bool we can use it as a predicate for the Where method, and also Select part is don’t needed.
Code: .Where(x => x.IsActive)